client/styles/_canvas.scss

```diff
@@ -207,34 +207,31 @@ body #app * {
 
 .minimap {
```

Comment:
Is there a reason for the minimal class to exist anymore? If so, it should be renamed to something else?